.github/teflo_terraform_workspace/main.tf

```diff
@@ -0,0 +1,31 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.60.0"
+    }
+  }
+}
+
+# Configure the AWS Provider
+provider "aws" {
```

Comment:
this provider key has nothing to do with the provider key that was removed in teflo correct? this is a separate teraform param?

Reply:
no